eng/pipelines/mirror-repos.yml

```
@@ -62,6 +62,10 @@ jobs:
               Azure/azure-sdk-for-android-pr:
               azure-sdk/azure-sdk-for-android:
 
+          Azure/azure-sdk-for-c:
+            TargetRepos:
+              Azure/azure-sdk-for-c-pr:
```

```suggestion
              Azure/azure-sdk-for-c-pr:
```
Might be worth adding the azure-sdk for for syncing as well.